samples/spring-eventsourced-counter/src/main/java/com/example/actions/Confirmed.java

```diff
@@ -1,5 +1,8 @@
 package com.example.actions;
 
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
```

Comment:
This has little to do with this PR. It was failing the serialization when running the integration test in this sample. Although the tests were passing.

Reply:
you can have a static instance 

```java
record Confirm() {
  public static Confirm instance = new Confirm();
}
```

A little bit more code, but no other annotation.

Reply:
:) that's simplicity 👍